### PR TITLE
Content: More Sokoban levels

### PR DESCRIPTION
**Pull request by Xaleth**
#### Summary
SUMMARY: Content "Add more sokoban levels"

#### Purpose of change
Have some NetHack flavor in the Sokoban minigame.

#### Describe the solution
Add eight levels to the Sokoban minigame from NetHack.

#### Describe alternatives you've considered
do nothing

#### Testing
It works.

#### Additional context
For legal purposes, the license has also been added.

Also, I slightly modified the levels to have a matching number of boulders and "pits." The first two added levels have one additional boulder in case a key got pressed one too many times.

### CODE REVIEW

**Comment by int-ua** [comment]
Can you add a description of the license? Is it compatible with Creative Commons that CDDA uses? Can it be placed or renamed to not imply it affects everything in `data/raw`?

**Comment by actual-nh** [comment]
> Can it be placed or renamed to not imply it affects everything in `data/raw`?

Call it "sokoban-nethack.license.txt"?

**Comment by Xaleth** [comment]
Should the copyright notice also be more specific? Like adding a description saying "Hence, the copyright." After saying the last 8 levels are from NetHack

File: data/raw/sokoban.txt [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ; Web Site: http://www.sourcecode.se/sokoban/levels.php
 ;
 ; Levels from the Game Boy.
+;
+; Copyright (c) 1998-1999 by Kevin Hugo
+; The last eight levels of this file are from NetHack.
```
Comment (actual-nh) [inline data/raw/sokoban.txt:8]:
```suggestion
; Levels 121-128 (currently the last 8 in the file) are from NetHack.
; Copyright (c) 1998-1999 by Kevin Hugo
; See sokoban-nethack.license.txt for the license for these levels.
```

**Comment by Maleclypse** [comment]
@ZhilkinSerg this looks like it should be able to merge into dev branch.  Let me know if this is helpful or not.

**Comment by ZhilkinSerg** [comment]
Apparently that was not merged to `master` branch - only to 0.F-dev

**Pull request merged**